apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -753,9 +753,9 @@ export class MarkdownDocumenter {
   }
 
   /**
-   * GENERATE PAGE: INTERFACE
+   * GENERATE PAGE: ITEM-CONTAINER
    */
-  private _writeInterfaceTables(output: DocSection, apiClass: ApiInterface): void {
+  private _writeItemContainerTables(output: DocSection, apiClass: ApiItemContainerMixin): void {
```

Comment:
Update the second parameter to be `apiInterfaceOrTypeAlias: ApiInterface|ApiTypeAlias`? (I know that the parameter name being incorrect is preexisting).